t/43-cli-api.t

```diff
@@ -159,67 +157,54 @@ subtest 'Simple request with authentication' => sub {
 };
 
 subtest 'HTTP features' => sub {
-    my ($stdout, @result) = capture_stdout sub { $api->run('--host', $host, 'test/pub/http') };
-    my $data = decode_json $stdout;
-    is $data->{method}, 'GET', 'GET request';
+    my $route = 'test/pub/http';
```

Comment:
Technically a path, and not a route.

Reply:
oh, ok. Variable names are important. Fixed that and updated.